templates/data_quality.html

```diff
@@ -11,8 +11,8 @@
     <!-- add bootstrap css file -->
             {% raw %}
 
-     <link rel="stylesheet" href="../css/bootstrap.min.css">
-     <link rel="stylesheet" href="../css/main.css">
+     <link rel="stylesheet" href="../../css/bootstrap.min.css">
+     <link rel="stylesheet" href="../../css/main.css">
```

Comment:
Better to use something like `{{ url_for("static", filename='css/main.css') }}` here.

Reply:
url_for only works with dynamic templates (I.e. with flask), right?

Reply:
Right, so if you're not using flask I guess it's OK as is (although you still might want to set a variable for `../..`, e.g. `root` and then use `{{ root }}` throughout so you only have to change things in one place).